pom.xml

```diff
@@ -59,9 +58,9 @@
             <artifactId>spring-boot-starter-validation</artifactId>
         </dependency>
         <dependency>
-            <groupId>junit</groupId>
-            <artifactId>junit</artifactId>
-            <version>${junit.version}</version>
+            <groupId>org.junit.vintage</groupId>
```

Comment:
This project now configured like `extensions-processor-api` for JUnit (which didn't require this change for the Java upgrade).